docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -138,11 +141,11 @@ Preparing For a Release
 
       git commit -am "Updated NEWS"
 
-#. Ensure the notes in `docs/advanced/whatsnew.rst <https://github.com/ClusterHQ/flocker/blob/master/docs/advanced/whatsnew.rst>`_ are up-to-date:
+#. Ensure the notes in `docs/releasenotes/index.rst <https://github.com/ClusterHQ/flocker/blob/master/docs/releasenotes/index.rst>`_ are up-to-date:
```

Comment:
I think that it helps because if the link is incorrect the link checker should find that and tell us. That will be useful when the link checker causes the build to fail.